wagtail/admin/viewsets/__init__.py

```diff
@@ -10,7 +10,11 @@ def __init__(self):
     def populate(self):
         for fn in hooks.get_hooks("register_admin_viewset"):
             viewset = fn()
-            self.register(viewset)
+            if isinstance(viewset, (list, tuple)):
+                for vs in viewset:
+                    self.register(vs)
+            else:
+                self.register(viewset)
```

Comment:
Not sure if it's worth adding a new hook just to allow registering multiple viewsets at a time, so I just do this.